src/category_theory/adjunction.lean

```diff
@@ -368,7 +368,20 @@ nat_iso.of_components (λ Y,
       naturality' := λ j j' f, begin
         erw [← adj.hom_equiv_naturality_left_symm, ← adj.hom_equiv_naturality_right_symm, t.naturality],
         dsimp, simp
-      end } } )
```

Comment:
This and the other change in this file don't look great...

Reply:
So... the root cause of this is that I tried moving `ext` later in the list of `tidy` tactics, in the hope that that might improve things. This helped some proofs in the new part of this PR, but broke some proofs about `adjunction`. I think I better separate out the two concerns. I'll put `tidy` back how it was, and investigate that change separately. Sorry for the noise.

Reply:
I've reversed these changes for now.